mep/books/views.py

```diff
@@ -17,6 +19,15 @@ class ItemList(LabeledPagesMixin, ListView):
     def get_queryset(self):
         return ItemSolrQuerySet().order_by('title')
 
+    def get_uri(self):
+        return absolutize_url(reverse('books:books-list'))
```

Comment:
I could imagine a default implementation that does this based on a class variable for the view route name, but don't know if it's common enough yet to warrant that.